client/src/hooks/Chat/useChatFunctions.ts

```diff
@@ -145,6 +147,7 @@ export default function useChatFunctions({
       parentMessageId = Constants.NO_PARENT;
       currentMessages = [];
       conversationId = null;
+      navigate('/c/new');
```

Comment:
I couldn't figure out why, but submissions don't work unless it's on this path. So, right before it's sent up I strip all the params from the URL. 

This was being done on page load in [client/src/hooks/Input/useQueryParams.ts](https://github.com/danny-avila/LibreChat/blob/main/client/src/hooks/Input/useQueryParams.ts#L226).